elements/pvpindicator.lua

```diff
@@ -71,7 +71,7 @@ local function Update(self, event, unit)
 
 		status = 'ffa'
 	elseif(factionGroup and factionGroup ~= 'Neutral' and UnitIsPVP(unit)) then
-		if(UnitIsMercenary(unit)) then
+		if(unit == 'player' and UnitIsMercenary(unit)) then
```

Comment:
Ask Blizzard T_T In merc mode `UnitFactionGroup` returns real faction group for player unit, but it works fine for other units, so there's no need to swap anything for them.